src/jit-format/jit-format.cs

```diff
@@ -104,7 +104,8 @@ private void SetPlatform()
                         }
                         else
                         {
-                            _os = match.Groups[1].Value.Trim();
+                            // Assuming anything other than Windows or OSX is a Linux flavor
+                            _os = "Linux";
```

Comment:
We're matching against OS Name, which varies based on OS. If I looked at OS platform, I could do that.